dace/config_schema.yml

```diff
@@ -111,7 +111,17 @@ required:
                 description: >
                    Allowing shadowing of variables in the code (reduces
                    exceptions to warnings when shadowing is encountered).
-                   
+
+            default_data_types:
+                type : str
+                default: Python default
```

Comment:
Maybe reduce to just "Python" and "C"? It's awkward to write this from the command line:
`DACE_default_data_types=Python\ default python3 samples/tizicoin.py`

Reply:
Agreed